Mathlib/Algebra/Order/Monoid/Unbundled/Basic.lean

```diff
@@ -283,6 +283,18 @@ theorem mul_right_cancel'' [MulRightReflectLE α] {a b c : α}
   haveI := mulRightMono_of_mulRightStrictMono α
   rw [le_antisymm_iff, eq_true (mul_le_mul' hac hbd), true_and, mul_le_mul_iff_of_ge hac hbd]
 
+@[to_additive] lemma mul_left_inj'' [MulRightStrictMono α] {a b c : α} {h : c ≤ b} :
```

Comment:
```suggestion
@[to_additive] lemma mul_left_inj_of_comparable [MulRightStrictMono α] {a b c : α} {h : b ≤ c \or c ≤ b} :
```
which hopefully wlog can dispatch quickly

Reply:
Thanks for the suggestion!  I updated the two lemmas.  Sadly, `wlog` is not imported in this file, though.